_tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md

```diff
@@ -8,9 +8,6 @@ grand_parent: Availability and recovery
 
 # Migrating to remote-backed storage
 
-This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/7986).    
-{: .warning}
-
 Introduced 2.14
```

Comment:
This label is confusing. Is this no longer an experimental feature? If so, then we should update label to Introduced 2.15. The subsequent text refers to 2.15.

Reply:
Yes, we are doing GA for it .